qt/python/mantidqt/widgets/samplelogs/model.py

```diff
@@ -151,9 +184,21 @@ def getItemModel(self):
         onto a QTableView
         """
 
-        def create_table_item(column, itemname, callable, *args):
+        def create_table_item(column, itemname, invalid_value_count, log_size, callable, *args):
             item = QStandardItem()
             item.setEditable(False)
+            #format if there is invalid data entries
+            if invalid_value_count == -1:
+                item.setData(QColor.fromHsv(0, 180, 255), Qt.BackgroundRole)
+                item.setToolTip("All of the values in the log are marked invalid, none of them are filtered.")
+            elif invalid_value_count > 0:
+                saturation = 10 + (170 * (invalid_value_count/(log_size+invalid_value_count)))
+                item.setData(QColor.fromHsv(0, saturation, 255), Qt.BackgroundRole)
+                tooltip_string = ("{count}/{size} of the values in the log {aux_verb} marked invalid, "
```

Comment:
Can we use [`f-strings`](https://realpython.com/python-f-strings/#f-strings-a-new-and-improved-way-to-format-strings-in-python) here? I find they generally improve the readability of this type of code.